[Documentation] HF MCP Server (#1938)
huggingface/hub-docs: docs/hub/agents.md

#### Getting Started

> Visit [huggingface.co/settings/mcp](https://huggingface.co/settings/mcp) to configure your MCP client and get started. Read the dedicated one‑page guide: [HF MCP Server](./hf-mcp-server).

Reviewer: ```suggestion
Visit [huggingface.co/settings/mcp](https://huggingface.co/settings/mcp) to configure your MCP client and get started with our [short guide](./hf-mcp-server).
```

Author: I have a slight preference for the former over this.
